package.json

```diff
@@ -16,7 +16,8 @@
     "packages/node",
     "packages/types",
     "packages/typescript",
-    "packages/utils"
+    "packages/utils",
+    "packages/eslint-config-typescript,"
```

Comment:
nit: this list was previously alphabetically sorted. do you feel that the eslint config should be first/last/fall in order?